src/rmodels.c

```diff
@@ -123,6 +123,8 @@
 //----------------------------------------------------------------------------------
 // Module specific Functions Declaration
 //----------------------------------------------------------------------------------
+static Vector3 *ToVector3(VoxVector3 *array);
```

Comment:
I'm afraid this should not  belong to `rmodels`, just assign the components individually when required.